systems/vector_scope.cc

```diff
@@ -0,0 +1,36 @@
+#include "drake/systems/framework/basic_vector.h"
+#include "drake/systems/framework/diagram_builder.h"
+#include "drake/systems/framework/leaf_system.h"
+#include "drake/systems/framework/framework_common.h"
+#include "drake/systems/framework/basic_vector.h"
+#include "drake/systems/framework/event.h"
+#include "systems/vector_scope.h"
+
+namespace dairlib {
+namespace systems {
+
+using drake::systems::BasicVector;
+using drake::systems::LeafSystem;
+using drake::systems::Context;
+using drake::systems::BasicVector;
+using drake::systems::kVectorValued;
+using drake::systems::EventStatus;
+
+VectorScope::VectorScope(int size, std::string debugMessage) : msg_(debugMessage) {
```

Comment:
This looks like a useful debugging tool. Two comments:

- Needs to be documented
- Publish rate should be configurable in constructor